src/components/utils/src/timer.cc

```diff
@@ -141,7 +127,7 @@ void timer::Timer::OnTimeout() {
   // Task locked by own lock because from this task in callback
   // we can call Stop of this timer and get DeadLock
   sync_primitives::AutoLock auto_lock(task_lock_);
-  DCHECK(task_);
+  DCHECK(task_.get());
   task_->run();
```

Comment:
@AGritsevich  I am not sure that it is good idea male OnTimeout function const.
It used not const run method of not const task_ pointer.